zio-http/src/main/scala/zio/http/middleware/Web.scala

```diff
@@ -37,6 +37,16 @@ private[zio] trait Web extends Cors with Csrf with Auth with HeaderModifier[Http
         } yield Patch.empty
     }
 
+  final def logResponseDefault: HttpMiddleware[Any, Nothing] =
+    interceptZIOPatch(req => Clock.nanoTime.map(start => (req.method, req.url, start))) {
+      case (response, (method, url, start)) =>
+        for {
+          end <- Clock.nanoTime
+          duration = (end - start) / 1000000
+          _ <- ZIO.log(s"${response.status.asJava.code()} ${method} ${url.encode} ${duration}ms")
+        } yield Patch.empty
+    }
```

Comment:
`debug` does this already isn't it.

Reply:
If this is a duplicate of `debug`, what do you think about using a new name, so as not to imply that the middleware only be used for debugging??

Reply:
> `debug` does this already isn't it.

It's very similar, but it strictly uses the Console, rather than the logging machinery. In addition to the rename that John proposed, we would also want to change the `debug` implementation to this version.

Reply:
Agree. Let's do that.